This repository lists papers in the biology field and related with the following key works 

(Do NOT need to be related to AI/Deep Learning. ONLY need to find pure biology papers on Genotype-Phenotype Associations. List 30 papers.)


Please follow the EXAMPLE

## Genotype-Phenotype Associations

- [Probing the genomic limits of de-extinction in the Christmas Island rat](https://pubmed.ncbi.nlm.nih.gov/35271794/)
  
   This paper examines the feasibility of de-extinction using the Christmas Island    rat (Rattus macleari) by mapping its genome to 
   the Norway brown rat (Rattus norvegicus). Despite recovering 95% of the genome, key genes related to immune response (MHC class I, 
   MAGEB18), olfaction (Vomeronasal receptor genes - Vom2r), and non-coding RNAs (snRNA, snoRNA, tRNA) were either incomplete or 
   entirely absent. The findings highlight that evolutionary divergence limits the accuracy of genome reconstruction, affecting 
  traits critical for survival.


1. **[Design and testing of a humanized porcine donor for xenotransplantation](https://www.nature.com/articles/s41586-023-06594-4)**

   Engineered pig kidneys with 69 edits worked long-term in monkey transplants, promising for human use.

1. **[Deconstructing the sources of genotype-phenotype associations in humans](https://www.science.org/doi/full/10.1126/science.aax3710)**

   2019，Genotype refers to variations in the human genome, phenotype refers to observable traits or disease outcomes.


1. **[Genetic effects on gene expression across human tissues](https://www.nature.com/articles/nature24277)**

   The GTEx project studies how genetic variation influences gene expression in 44 human tissues. Most genes are affected by local genetic variation, and some have inter-chromosomal effects. The study explores tissue-specific expression, genetic regulation, and disease-associated genes, providing insights into gene regulation and disease mechanisms.


1. **[Genetic basis of ecologically relevant body shape variation among four genera of cichlid fishe](https://onlinelibrary.wiley.com/doi/pdfdirect/10.1111/mec.16977)**

   The study identifies 34 QTLs affecting body shape variations in cichlid fishes from East African Rift Lakes, revealing distinct genetic mechanisms.


1. **[From GWAS to signal validation: An approach for estimating genetic effects while preserving genomic context](https://pmc.ncbi.nlm.nih.gov/articles/PMC10028994/)**

   New method validates SNP-level genotype-phenotype associations in outbred populations using synthetic crosses, tested on Drosophila for longevity impacts of a natural cis-eQTL.


1. **[Heterogeneity of genetic architecture of body size traits in a free-living population](https://onlinelibrary.wiley.com/doi/full/10.1111/mec.13146)**

   The study investigates the genetic architecture of body size traits in a free-living population of Soay sheep using 37,037 SNPs. It finds that jaw and weight exhibit polygenic inheritance, while leg length is influenced by two major SNPs. It also suggests that alleles for shorter legs entered the population through historical admixture.


1. **[Providing biological context for GWAS results using eQTL regulatory and co-expression networks in Populus](https://nph.onlinelibrary.wiley.com/doi/10.1111/nph.20026)**

   This study integrates GWAS, eQTL analysis, and co-expression networks to provide biological context for trait-associated loci in Populus trichocarpa. It identifies three candidate genes involved in lignocellulose biosynthesis, including Potri.005G116800/VND1 (a key regulator of secondary cell wall formation) and two novel candidates. This approach offers a framework for uncovering gene function and regulatory networks in tree species. 

1. **[Polygenic basis for adaptive morphological variation in a threatened Aotearoa | New Zealand bird, the hihi (Notiomystis cincta)](https://royalsocietypublishing.org/doi/full/10.1098/rspb.2020.0948)**

   This study examines the genetic basis of three adaptive morphological traits in the threatened New Zealand bird Notiomystis cincta. Using 523 individuals and 39,699 SNPs, it finds that the traits have low heritability and a polygenic basis, with many small-effect loci contributing to trait variation. GWAS suggests that a locus in HEY2 may be associated with tarsus length. Due to the species’ small population size and polygenic inheritance, its ability to adapt to selection is likely constrained.


1. **[Genome-Wide Association Mapping of Phenotypic Traits Subject to a Range of Intensities of Natural Selection in Timema cristinae](https://www.journals.uchicago.edu/doi/full/10.1086/675497?casa_token=PMHeQg24764AAAAA%3AwTrtIkO8vMhxbEz1hxjbZiWldsdN6BuSUD2M9zVR2kaI7aOHDcULs408IWNaqSq52wrE8cD6SGvG)**

   This study examines the genetic architecture of adaptive traits under different selection intensities in Timema cristinae. Using GWAS with 211,004 SNPs and measurements of 10 phenotypic traits, it finds that some SNPs significantly contribute to trait variation. Both major and minor effect loci are identified, but no direct correlation is found between trait divergence in nature and their genetic architecture. This research is a first step in identifying loci associated with adaptation in T. cristinae, with future studies focusing on population differentiation and selection response of these SNPs.

1. **[Independent and Joint-GWAS for growth traits in Eucalyptus by
assembling genome-wide data for 3373 individuals across four
breeding populations](https://nph.onlinelibrary.wiley.com/doi/pdfdirect/10.1111/nph.15449)**

   This study conducted independent and joint-GWAS on 3373 Eucalyptus individuals across four breeding populations, using multiple models to identify genetic associations with growth traits. While single-SNP GWAS had limited power, gene-based Joint-GWAS identified nine significant genes related to tree height. Whole-genome regression captured 64-89% of the pedigree-based heritability. The study also validated associations in forest trees and identified key genes involved in cell wall biosynthesis and lignification. These results suggest that Joint-GWAS can improve tree breeding by identifying useful genetic markers.

1. **[The Genetic Architecture of Maize Flowering Time](https://www.science.org/doi/10.1126/science.1174276)**

   Maize flowering time is controlled by many small-effect QTLs rather than a single large-effect locus. A study of 5000 inbred lines across eight environments found that allelic effects vary among founder lines, with no major epistasis or environmental interactions. A simple additive model effectively predicts flowering time, differing from the genetic architecture of rice and Arabidopsis.


1. **[Combining genome-wide association study and FST-based approaches to identify targets of Borrelia-mediated selection in natural rodent hosts](https://onlinelibrary.wiley.com/doi/full/10.1111/mec.15410?casa_token=iyeRaaHD4LYAAAAA%3AknCigP3eDxMeEHERMfzIK-4qChD5wtjcr0vi_J4NVeVEG7sFCgRHGgfhpR54rpkokfrIaVbeNzaXyzA)**

   This study combines GWAS and FST-based methods to identify genomic regions associated with Borrelia afzelii resistance or susceptibility in Myodes glareolus, a natural rodent host of the Lyme disease-causing spirochete. Using SNP data from genotyping-by-sequencing (GBS), four key genes (Slc26a4, Tns3, Wscd1, and Espnl) were identified as significantly associated with Borrelia infection status. These genes have known immune-related functions in rodents, making them promising candidates for studying host adaptation to bacterial infections. The approach used here is applicable to other wildlife populations for identifying genetic variants underlying disease resistance and other ecologically relevant traits.


1. **[Combining paired analytical metabolomics and common garden trial to study the metabolism and gene variation of Ginkgo biloba L. cultivated varieties](https://pubs.rsc.org/en/content/articlehtml/2017/sc/c7ra06229j)**

   This study integrates metabolomics and a common garden trial to explore genetic variation in flavonoid biosynthesis among 30 cultivated Ginkgo biloba varieties. Using UHPLC-QQQ MS/MS, 36 metabolites were quantified, linking SNP variation in enzyme-related genes to metabolic differences. Findings indicate that genetic polymorphisms influence flavonoid synthesis, although heritability is subtle. This method provides an accurate, low-sample, and efficient approach for phenotype-genotype association studies and is applicable to artificial breeding assessments.



1. **[Aphid wing dimorphisms: linking environmental and genetic control of trait variation](https://royalsocietypublishing.org/doi/full/10.1098/rstb.2009.0255)**

   This study reviews the molecular mechanisms underlying aphid wing dimorphism, where wing formation is controlled by either genetic or environmental factors. Some aphid species alternate between polyphenic (environmental) and polymorphic (genetic) wing determination in their life cycle. Understanding genetically controlled polymorphisms may provide insight into the molecular basis of polyphenic variation and facilitate comparisons between genetic and environmental control mechanisms of similar traits.



1. **[Natural gene expression variation studies in yeast](https://onlinelibrary.wiley.com/doi/full/10.1002/yea.3210?casa_token=yQZ3uK5PPgEAAAAA%3AoTdrtN0vJNKVxOSpOzlusKdsPElc90JrhT2foVbXeGH5f5YxypiiUHH_jCtpmJ-75k8nHXrITzjvDrA)**

   This study reviews recent advances in yeast genomics, focusing on natural gene expression variation and its impact on genotype-phenotype associations. By analyzing mRNA abundance, allele-specific expression, and translation efficiency, the study explores how genetic variants co-evolve within genomes. It also highlights contradictions between inter- and intra-specific studies, likely due to genetic divergence. Future research will benefit from improved genome annotations and next-generation sequencing technologies for a more detailed understanding of genotype-phenotype mapping.


1. **[Divergent Allometric Trajectories in Gene Expression and Coexpression Produce Species Differences in Sympatrically Speciating Midas Cichlid Fish](https://academic.oup.com/gbe/article/11/6/1644/5498148)**

   This study investigates gene expression and coexpression differences in sympatrically speciating Midas cichlid fish using RNA-Seq. By analyzing two developmental stages, it identifies coexpression gene modules that differ between benthic and limnetic forms. The results indicate that gene coexpression divergence starts as early as one day post-hatch and becomes more pronounced as the fish grow. This study highlights the role of coexpressed gene networks, rather than single genes, in driving morphological divergence between ecotypes.

1. **[Identification of Genetic Variants Underlying Anxiety and Multiple Sclerosis in Heterogeneous Stock Rats](https://www.scirp.org/html/2-1390190_46654.htm)**
   
   This study used genetic mapping and sequencing in heterogeneous stock rats to identify genotype-phenotype associations in anxiety and multiple sclerosis. Key genes include Ctnnd2, linked to anxiety-related response latency, ABCB10, affecting red blood cell volume, and Btnl2 and RT1-Db1 in the MHC region, associated with autoimmune encephalomyelitis. The findings demonstrate the power of genome-wide association and sequencing in uncovering genetic influences on complex traits.

1. **[A Single Amino Acid Mutation Contributes to Adaptive Beach Mouse Color Pattern](https://www.science.org/doi/10.1126/science.1126121)**

   This study identified a Mc1r mutation in beach mice that reduces receptor function, contributing to their adaptive light coloration for crypsis. Genetic analysis showed that Mc1r variation explains a significant portion of pigmentation differences, but similar coloration in Atlantic coast mice arises from different mechanisms. The findings demonstrate how a single amino acid change in a pigmentation gene drives adaptive evolution in natural populations.

1. **[The genetic basis of adaptive melanism in pocket mice](https://www.pnas.org/doi/abs/10.1073/pnas.0431157100)**

   This study identified four Mc1r mutations responsible for adaptive melanism in lava-dwelling rock pocket mice, aiding predator avoidance. A separate melanic population lacked Mc1r mutations, suggesting independent genetic pathways for similar adaptation. The findings highlight convergent evolution through different molecular mechanisms.

1. **[Genetic and developmental basis for fin shape variation in African cichlid fishes](https://onlinelibrary.wiley.com/doi/abs/10.1111/mec.13905?casa_token=wxrZhZM09HwAAAAA:G6RvMCUy0SpcFZEGlGo0BcEEwS8fQ4MvvDkar3Pa1Wlxp_-QLypRM8E1CTWg6GL6grXalUG_SiPfkws)**
   
   This study identified wnt7aa and col1a1 as key genes influencing pectoral fin shape variation in African cichlids. wnt7aa affects fin elongation, while col1a1 regulates the number of fin ray elements. Genetic mapping and genome scans showed that fin shape evolves independently of body and craniofacial traits, likely due to selective pressures rather than genetic linkage. Experimental validation confirmed these genes’ roles in fin development, highlighting their contribution to morphological divergence in adaptive radiation.

1. **[Candidate Gene Analysis Suggests Untapped Genetic Complexity in Melanin-Based Pigmentation in Bird](https://academic.oup.com/jhered/article-abstract/107/4/327/2622968?redirectedFrom=fulltext)**

   This study identified Corin and TYRP1 as key genes influencing melanin-based plumage variation in the Réunion grey white-eye, with allele shifts between lowland and highland forms suggesting balancing selection in the polymorphic highland population. In contrast, Agouti, a common pigmentation gene, showed no association with color forms. These findings highlight unexplored genetic complexity in avian pigmentation beyond traditionally studied loci.

1. **[A Val85Met Mutation in Melanocortin-1 Receptor Is Associated with Reductions in Eumelanic Pigmentation and Cell Surface Expression in Domestic Rock Pigeons (Columba livia)
](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0074475)**

   This study identified a Val85Met mutation associated with reduced eumelanin pigmentation in domestic rock pigeons. Functional assays showed that this mutation decreases receptor activity and cell surface expression, leading to pheomelanism instead of eumelanism.

1. **[Avian Coloration Genetics: Recent Advances and Emerging Questions](https://academic.oup.com/jhered/article/112/5/395/6272461)**

   Bird coloration results from interactions between pigment genes (carotenoids for red, yellow, orange; psittacofulvins in parrots) and structural genes that manipulate light at the nanoscale. Recent advances suggest co-expression or co-regulation of these genes influences mixed-color traits, while structural color genetics remain underexplored. Genetic insights continue to drive discoveries linking color production to vision and speciation.

1. **[Genetics, development and evolution of adaptive pigmentation in vertebrates
](https://www.nature.com/articles/6800861)**

   ​This article examines how pigmentation has served as a model for understanding the genetic, developmental, and evolutionary mechanisms in vertebrates. It highlights that mutations in specific genes, such as Mc1r and Agouti, can lead to variations in melanin production, resulting in diverse coloration patterns. These genetic changes often correlate with environmental adaptations, providing insights into the molecular basis of organismal diversity and the evolutionary process.   

1. **[Transcriptomics of colour patterning and coloration shifts in crows](https://onlinelibrary.wiley.com/doi/10.1111/mec.13353)**

   This study reveals that eumelanin-based plumage differences between black carrion crows and grey-coated hooded crows result from gene expression variation. ASIP, CORIN, and ALDH6 regulate regional pigmentation through prepatterning, while HPGDS, NDP, and RASGRF1, under MITF control, drive color-specific expression. These findings suggest that stable prepatterns, modified by regulatory changes, shape recurrent color patterns across vertebrates.

1. **[Extensive hybridization reveals multiple coloration genes underlying a complex plumage phenotype](https://royalsocietypublishing.org/doi/10.1098/rspb.2020.1805)**

   ​In North American flickers, extensive hybridization between yellow-shafted and red-shafted forms has revealed that multiple genes influence their distinct plumage coloration. Analysis of approximately 7.25 million genome-wide SNPs identified 368 SNPs significantly associated with four of six plumage patches. Notably, the gene CYP2J19, known to affect color transitions in other birds, is strongly linked to the yellow versus red differences in wing and tail feathers.
 
1. **[Admixture mapping in a hybrid zone reveals loci associated with avian feather coloration](https://royalsocietypublishing.org/doi/10.1098/rspb.2017.1106)**

   In a hybrid zone between Audubon's and myrtle warblers, researchers identified multiple genomic regions associated with plumage coloration traits. Notably, the SCARF2 gene was linked to carotenoid-based throat color differences, while a specific region on chromosome 20 was associated with melanin-based eye line and eye spot coloration. These findings highlight the genetic basis of feather coloration in these species.

1. **[Differential gene regulation underlies variation in melanic plumage coloration in the dark-eyed junco (Junco hyemalis)](https://onlinelibrary.wiley.com/doi/10.1111/mec.14878)**

   Melanic plumage differences between slate-colored and Oregon juncos result from differential gene regulation rather than coding mutations. Lighter feathers show downregulation of PMEL, TYR, TYRP1, OCA2, MLANA (melanin synthesis) and upregulation of ASIP. Feathers from different body regions exhibit distinct HOX and WNT gene expression patterns, while ASIP, MFSD12, KCNJ13, and HAND2 regulate color differences between subspecies. These findings highlight how gene regulation, rather than sequence variation, drives rapid plumage diversification.

1. **[Genome-wide association mapping identifies the genetic basis of discrete and quantitative variation in sexual weaponry in a wild sheep population](https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2011.05076.x)**

   In Soay sheep, horn morphology is controlled by RXFP2, an autosomal gene that determines whether individuals develop normal horns, scurs, or no horns. A genome-wide association study identified RXFP2 as the primary locus influencing this trait, explaining up to 76% of the additive genetic variation in horn size among normal-horned males. This study establishes a genetic basis for horn polymorphism and provides insights into the persistence of this trait despite natural selection pressures.

1. **[Association mapping of morphological traits in wild and captive zebra finches: reliable within, but not between populations](https://onlinelibrary.wiley.com/doi/10.1111/mec.14009)**

   In zebra finches, association mapping of morphological traits identified 672 SNPs within candidate genes from previously mapped QTL regions. While genotype-phenotype associations were weak in wild populations due to short linkage disequilibrium (LD) blocks, captive populations with larger LD blocks showed strong and repeatable associations. However, effect direction and magnitude varied across populations, suggesting that these SNPs are linked to but not the causal QTN. The study highlights how founder effects influence LD patterns, affecting trait mapping reliability across populations.

1. **[Comparative genomics of two Empidonax flycatchers reveal candidate genes for bird song production](https://academic.oup.com/evolut/article/77/8/1818/7186177)**

   In Empidonax flycatchers, genomic comparisons identified six highly differentiated regions linked to differences in innate vocal signals. Among 67 genes in these regions, three are associated with language impairment and dyslexia in humans, while 18 show differential expression in avian song nuclei. These findings suggest a genetic basis for species-specific song production and provide candidate loci for further studies on vocalization in birds.

1. **[The genetics of behaviour and other adaptive traits in nine-spined sticklebacks (Pungitius pungitius)](https://www.utupub.fi/bitstream/handle/10024/92417/AnnalesAII283LaineDISS.pdf?sequence=2)**

   In nine-spined sticklebacks, Pitx1 was identified as a key gene influencing pelvic structure, confirmed through QTL mapping. Behaviour and body size exhibited a more complex genetic basis, with multiple QTLs linked to these traits. Heterozygosity correlated with behaviour in a population-dependent manner, with osmoregulation-related gene markers showing strong behavioural associations. These findings suggest pleiotropic effects in QTL regions shaping morphology and behaviour.

